package.json

```
@@ -7,22 +7,27 @@
   "main": "lib/cjs/index.js",
   "module": "lib/esm/index.js",
   "types": "lib/cjs/index.d.ts",
+  "packageManager": "pnpm@7.25.0",
+  "engines": {
+    "node": "18.x",
```

```suggestion
    "node": ">=18",
```

We can assume newer versions of node will work (we are testing 20 after all)